### ISSUE TITLE
Build latest version with different CUDA

### ISSUE THREAD

**Issue by Destroy666x** [royshil/obs-localvocal#81]
I noticed there's now some `CPU_OR_CUDA` parameter which doesn't let me build it with CUDA 12.3 which I have installed and tht worked fine previously, e.g. 
```
 .github/scripts/Build-Windows.ps1 -Target x64 -ExtraCmakeArgs '-D','LOCALVOCAL_WITH_CUDA=ON','-D',"CUDA_TOOLKIT_ROOT_DIR='C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.3'"
```
. Any reason of this limitation?

**Comment by royshil** [comment]
@Destroy666x I've updated the build instructions now take a look in the readme, it's very simple

Additionally you don't need to build from source with cuda anymore because I prebuild it for you for cuda 12 and 11, so just download and install

**Comment by Destroy666x** [comment]
Yea, I've seen. But I'm asking for building for specific version. Is it not possible anymore? I think I see now that it ues some prebuilt binaries

**Comment by royshil** [comment]
@Destroy666x yes it's possible, but now i'm building whisper separately since the CUDA builds on CI are easily 20-25 minutes which is way too much. 
you can still build localvocal and it will fetch the prebuilt whisper for cuda from https://github.com/occ-ai/occ-ai-dep-whispercpp

**Comment by Ivan2getdmodelz** [comment]
Related to this to topic, I'm interested in pointing localvocal to whisper.cpp with cuda enabled on ubuntu.  Do I need to set an environment variable to enable it to run the models on a GPU?

**Comment by Destroy666x** [comment]
So is it possible now again @Tabby ?

**Comment by Tabby** [comment]
It is on Linux >.>

I was just being a bit ruthless at closing old issues and as there was no reply to Roy's last comment on this issue I assumed that meant it was resolved. If you still want the ability to build with any CUDA version I can reopen it and try to get the source builds working on other platforms too

Out of curiosity, why do you want to be able to build it with specific versions of CUDA?

**Comment by Destroy666x** [comment]
Because:
- it allows easy reverting if some version is e.g. more laggy
- it wastes less disk space to just use CUDA available in the system
